plugins/CoreHome/vue/src/Menudropdown/Menudropdown.vue

```diff
@@ -29,6 +29,7 @@
         <input
           type="text"
           v-model="searchTerm"
+          v-focus-if="{ focusIf: showItems }"
```

Comment:
Should that give the search box a focus when the menu is opened? For me that doesn't work

Reply:
I think that should work, but not perfect

```suggestion
         autofocus
```

Reply:
> I think that should work, but not perfect

That could work here, but unfortunately we need the directive to continue working for backwards compatibility with existing anguarjs code.

Looks like Vue 3 directive values are static, once set they never change. I changed it instead to use the directive argument, which is supplied like `v-focus-if:[argument]`. It looks weirder, but it's the only way I could get it to work.

Reply:
@diosmosis somehow the showItems is passed as false. I guess the internal times have to be before the if blind value.

Reply:
@peterhashair it's always false, the value in the directive can't change after it's first set apparently. Only the argument is allowed to be dynamic according to https://v3.vuejs.org/guide/custom-directive.html.